### ISSUE TITLE
update aws sdk to ga version

### ISSUE THREAD

**Issue by ymgyt** [ymgyt/tracing-cloudwatch#29]
https://aws.amazon.com/blogs/developer/announcing-general-availability-of-the-aws-sdk-for-rust/

**Comment by calumpeak** [comment]
I'm assuming the upgrade is needed causing a client type mismatch when using?

```
aws-config = { version = "1.0.3", features = ["behavior-version-latest"] }
aws-sdk-cloudwatchlogs = "1.4.0"
```

```rust
error[E0277]: the trait bound `aws_sdk_cloudwatchlogs::Client: CloudWatchClient` is not satisfied
  --> src/main.rs:28:17
   |
27 |             tracing_cloudwatch::layer().with_client(
   |                                         ----------- required by a bound introduced by this call
28 |                 client,
   |                 ^^^^^^ the trait `CloudWatchClient` is not implemented for `aws_sdk_cloudwatchlogs::Client`
   |
   = help: the following other types implement trait `CloudWatchClient`:
             tracing_cloudwatch::client::NoopClient
             aws_sdk_cloudwatchlogs::client::Client
note: required by a bound in `CloudWatchLayer::<S, D>::with_client`
  --> /Users/calumpeak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tracing-cloudwatch-0.1.4/src/layer.rs:65:17
   |
59 |     pub fn with_client<Client>(
   |            ----------- required by a bound in this associated function
...
65 |         Client: CloudWatchClient + Send + Sync + 'static,
   |                 ^^^^^^^^^^^^^^^^ required by this bound in `CloudWatchLayer::<S, D>::with_client`

```

**Comment by ymgyt** [comment]
@calumpeak 
Sorry for the delay in response, we have released v0.1.5 and updated the version of aws-sdk-cloudwatchlogs and aws-config to ver 1
